RELEASE-NOTE.md

```diff
@@ -1,3 +1,28 @@
+3.5.4 (2021.03.10)
+==================
+- Changes
+    - Change the HttpClient of the test script to apache http components base
+        * Support modern HTTP specification
+        * Provide easy-to-use APIs
+    - Look up controller ip from dns when agent re-tries to connect to controller
+    - Support dynamic modification of script template
+        * When creating a script, will refer to under the NGRINDER_HOME/script_template
+    - Provide a test configuration to maintain the used connection on each test run
```

Comment:
Provide the connection reset option in test configuration page.
- If you turn it on, it tries to reuse connection as much as possible.